### DISCUSSION TITLE
URL query parameters

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#832] [Q&A]
hey, could someone list all of the url parameters, such as
- ?profile= (profile)
- ?use_bochs_bios=
- ?hda.url=<?>
- ?cdrom.url=<https://url-to.iso/file>
- fda.url=
- ?c= (command)

**Comment by copy** [comment]
You can search for `query_args` in https://github.com/copy/v86/blob/master/src/browser/main.js

**Comment by CorruptionVR** [comment]
hey, please tell me if i got any wrong, and the ones that I don't understand are `?use_boch_bios`, because I don't know what it does, and `?hda.url`, and `?fda.url` because I don't know what the filetypes are.

edit: ok, so, going off of context, I'm guessing that hda means hard drive disk image, and fda means floppy disk image? if not, please tell me what they are.
